raiden/tests/utils/smoketest.py

```diff
@@ -69,13 +79,13 @@
 
 if TYPE_CHECKING:
     # pylint: disable=unused-import
-    from raiden.network.transport.matrix import ParsedURL  # noqa: F401
+    from raiden.tests.utils.transport import ParsedURL  # noqa: F401
 
 # the smoketest will assert that a different endpoint got successfully registered
-TEST_PARTNER_ADDRESS = "2" * 40
-TEST_DEPOSIT_AMOUNT = 5
+TEST_PARTNER_ADDRESS = Address(to_canonical_address(b"2" * 40))
```

Comment:
`to_canonical_address` already returns an `Address`, so no need for the cast. And it shouldn't be a bytestring.